app/metrics.yaml

```diff
@@ -83,3 +83,168 @@ searches:
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  activated:
+    type: counter
+    description: >
+      Number of activated tabs
+    send_in_pings:
+      - session_end
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+firefox_account:
+  sign_in:
+    type: event
+    description: >
+      A user pressed the sign in button on the sync account page
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_success:
+    type: event
+    description: >
+      A user was successfully signed in to FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_error:
+    type: event
+    description: >
+      An error happened during the login flow and the user couldn't sign in to FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_out:
+    type: event
+    description: >
+      A user pressed the sign out button on the sync account page and was successfully signed out of FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  bookmarks_sync_status:
+    type: boolean
+    description: >
+      Status of the bookmarks sync engine
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  history_sync_status:
+    type: boolean
+    description: >
+      Status of the history sync engine
```

Comment:
This description is a bit unclear to me. This is a `boolean` type, how does it capture the engine status? What does `true` mean? What does `false` mean? Can you please add it to the description?

Reply:
Added info in the previous comment. Also made the metrics description more verbose.